stack.yaml

```diff
@@ -1,8 +1,9 @@
-resolver: lts-13.24
+resolver: lts-13.26
+allow-newer: true
```

Comment:
Is this allow-newer needed?

Reply:
it was there before... We could try removing it in another PR maybe?